src/components/forms/inputField/InputField.tsx

```diff
@@ -56,6 +60,8 @@ const InputField = ({
         autoComplete={autoComplete}
         autoCorrect={autoCorrect}
         type={type}
+        max={max}
+        min={min}
         className={styles.input}
         spellCheck={spellCheck}
         value={value}
```

Comment:
Should note that `max` and `min` are relevant for more than just `number`, specifically `date`, `month`, `week`, `time`, `datetime-local` and `range`. Since this is a general component, we should not assume the only use case is for numbers.